wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/templates/sample/Robot.java

```diff
@@ -130,9 +137,10 @@ public void autonomous() {
 	public void operatorControl() {
 		m_robotDrive.setSafetyEnabled(true);
 		while (isOperatorControl() && isEnabled()) {
-			// drive arcade style
+			// Drive arcade style
 			m_robotDrive.arcadeDrive(m_stick.getX(), m_stick.getY());
-			// wait for a motor update time
+
+			// Wait for a motor update time
```

Comment:
This is a confusing comment

Reply:
Agreed.

Reply:
Can we change it to "Allow the rest of the robot to update"? Cause idk what a motor update time is.

Reply:
To be specific, it's yielding to the network communication process to avoid consuming all the CPU time. Seems a bit low-level for the average rookie.

Reply:
My message gives it some meaning while keeping it simple. It teaches them that the rest of the robot does need to update and you need to give that delay.